### PR TITLE
Minor UI fixes

### LINKED ISSUE #250: Change the default numberOfAdults from 2 to 1
The default numberOfAdults does not match up with what is being displayed in the drop down in the edit details page

### PR DESCRIPTION
**Pull request by jackcarlisle**
#250 
* fixes default value for numberOfAdults
* fixes the destination page info so that it doesn't render tags #246

### CODE REVIEW

File: src/reducers/travel-info.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
 
 export const initialState = {
   numberOfChildren: '0',
-  numberOfAdults: '2',
+  numberOfAdults: '1',
```
Comment (nelsonic) [inline src/reducers/travel-info.js]: @jackcarlisle `'1'` adult ?

**Comment by nelsonic** [comment]
@jackcarlisle is there an *issue* for the `numberOfAdults` initial state change?

**Comment by jackcarlisle** [comment]
@nelsonic it was something that Roy mentioned in the stand-up this morning but I haven't made an issue for it

**Comment by nelsonic** [comment]
@jackcarlisle thanks for clarifying that. 👍

**Pull request merged**